src/app/components/user/connection/connection.component.ts

```diff
@@ -1,29 +1,59 @@
 import { Component, OnInit } from '@angular/core';
 import { FormBuilder, Validators, FormGroup } from '@angular/forms';
 import { UserService } from 'src/app/services/user.service';
+import { UsersApiService } from 'src/app/services/users-api.service';
 import { User } from 'src/app/shared/interfaces/user';
+import { confirmPasswordValidator, emailValidator, passwordValidator } from 'src/app/shared/validators/validators';
+import { SnackBarService } from 'src/app/services/snack-bar.service';
 
 @Component({
   selector: 'app-user-connection',
   templateUrl: './connection.component.html',
   styleUrls: ['./connection.component.scss']
 })
-export class ConnectionComponent implements OnInit{
+export class ConnectionComponent implements OnInit {
```

Comment:
optional : I think onInits interface is not required anymore in newest versions of Angular

Reply:
Done.